lib/rubocop/cop/style/dot_position.rb

```diff
@@ -71,6 +77,10 @@ def autocorrect(node)
             end
           end
         end
+
+        def comment_lines
+          processed_source.comments.map(&:location).map(&:line)
```

Comment:
There is no need to call `map` twice hear. This can be handled in one pass instead of two

```ruby
processed_source.comments.map { |comment| comment.location.line }
```